code/src/java/pcgen/gui2/csheet/CharacterSheetPanel.java

```diff
@@ -103,17 +103,13 @@ public CharacterSheetPanel()
 
 		// KAW TODO rewrite to use StatusWorker and PCGenTask for better progress display
 
-		refreshService = Executors.newSingleThreadExecutor(new ThreadFactory()
+		refreshService = Executors.newSingleThreadExecutor(r ->
 		{
-			@Override
-			public Thread newThread(Runnable r)
-			{
-				Thread thread = new Thread(r, "Character-Sheet-Refresher-Thread");
-				thread.setDaemon(true);
-				thread.setPriority(Thread.NORM_PRIORITY);
-				return thread;
-			}
-		});
+            Thread thread = new Thread(r, "Character-Sheet-Refresher-Thread");
```

Comment:
found the setting: it was set to use spaces instead of the tab character. Future PRs should be fixed. Please continue to point out other style changes that I should fix. Ideally all of this could be stored in the repo so future contributors have an easier time :)